### PR TITLE
Removing entities improvements

### PR DESCRIPTION
**Pull request by Aggror**
# PR Details
Updated removing entities scene. renamed scene folder.

## Types of changes
- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```
Comment (xen2) [inline _config.yml:1]: File added by mistake?

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```
Comment (xen2) [inline docs/_config.yml:1]: File added by mistake?

File: _config.yml (hunk 1 above)
Comment (Aggror) [reply _config.yml:1]: Yeah I was toying around with Github pages, to see if we could generate the Documentation as github pages. That way we could include the docs in main repo. But it was a failed attempt. In the mean time my config file clutted the branch it seems.

**Comment by Aggror** [comment]
I will have to check out why there are conflicts. Will do later this week.

**Comment by Basewq** [comment]
> 
> 
> I will have to check out why there are conflicts. Will do later this week.

My bad, it's probably because of my pull where I fixed some typos in files you moving.
https://github.com/stride3d/stride/pull/716/files

**Pull request closed without merging**